src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsForm.tag

```diff
@@ -78,6 +83,66 @@
           </div>
         </div>
         <br class="hidden-xs">
+        <div class="row">
+          <div class="col-sm-12">
+            <div class="form-group">
+              <h5 class="col-sm-2">
+                <label class="control-label" for="<%= Const.ParamsNames.COURSE_NAME %>">
+                  Course name
+                </label>
+              </h5>
+              <div class="col-sm-10 col-md-8">
+                <c:choose>
+                  <c:when test="${fsForm.courseIdEditable}">
+                    <c:forEach items="${courseAttributes}" var="attributes">
+                      <input class="course-name-data" id="${fn:escapeXml(attributes.id)}"
+                          type="hidden" value="${fn:escapeXml(attributes.name)}">
+                    </c:forEach>
+                    <h5 id="<%= Const.ParamsNames.COURSE_NAME %>" class="form-control-static font-weight-normal">
+                    </h5>
+                  </c:when>
+                  <c:otherwise>
+                    <h5 id="<%= Const.ParamsNames.COURSE_NAME %>" class="form-control-static font-weight-normal">
+                      ${courseName}
+                    </h5>
+                  </c:otherwise>
+                </c:choose>
+              </div>
+            </div>
+          </div>
+        </div>
+        <br class="hidden-xs">
+        <c:choose>
+          <c:when test="${!fsForm.courseIdEditable}">
+            <div class="row">
+              <div class="col-sm-12 col-md-6">
+                <div class="form-group">
+                  <h5 class="col-sm-2 col-md-4">
+                    <label class="control-label">
+                      Submission Status
+                    </label>
+                  </h5>
+                  <div class="col-sm-10 col-md-8">
+                    <h5>${PageData.getInstructorSubmissionStatusForFeedbackSession(fsAttributes)}</h5>
```

Comment:
Let's not add a dependency to static methods in here. Define this in the corresponding template object, e.g. `FeedbackSessionsForm` instead.